src/Apps/Artist/Routes/Overview/Components/CurrentEvent.tsx

```diff
@@ -59,9 +59,11 @@ export class LargeCurrentEvent extends Component<CurrentEventProps> {
         onClick={this.handleClick.bind(this)}
       >
         <Flex flexDirection="column">
-          <Box width="100%" height="auto">
-            <Image src={image.resized.url} width="100%" mb={1} />
-          </Box>
+          {image && (
```

Comment:
Was seeing this in Sentry, there can be a published current sale or show w/o an image (ie- there's nothing defending that at the database level). So since it's a possibility (on staging), and theoretically, wound up just being safe.